src/sphinx_theme_builder/_internal/cli/serve.py

```diff
@@ -84,6 +92,7 @@ def run(
                 # sphinx-autobuild flags
                 f"--watch={os.fsdecode(project.source_path)}",
                 f"--re-ignore=({'|'.join(map(re.escape, project.compiled_assets))})",
+                f"{f'--host={host}' if host else ''}",
```

Comment:
Let's add this via an append, instead of a somewhat hacky-mess of f-strings? That way this will be handled in a manner similar to the other optional arguments.